phys2bids/tests/test_physio_obj.py

```diff
@@ -101,7 +101,7 @@ def test_BlueprintInput():
 
     # Tests rename_channels
     new_names = ['trigger', 'time', 'lindt']
-    blueprint_in.rename_channels(new_names.copy())
+    blueprint_in.rename_channels(new_names)
     assert blueprint_in.ch_name == ['time', 'trigger', 'lindt']
```

Comment:
```suggestion
    assert blueprint_in.ch_name == ['time', 'trigger', 'lindt']
    assert test_index[0] is not test_trigger
```

If it's not storing the original array then this should pass.